src/promptflow/promptflow/executor/_service/contracts/base_request.py

```diff
@@ -0,0 +1,13 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Any, Mapping, Optional
+
+from pydantic import BaseModel
+
+
+class BaseRequest(BaseModel):
```

Comment:
I guess we could also put it in execution_request

Reply:
`BaseExecutionRequest` is only used to run flow and single node and tool related apis may also use this, so we make this class as the base class and other requests can directly inherit it.